packages/react-native/src/primitives/Radio/types.ts

```diff
@@ -0,0 +1,38 @@
+import { StyleProp, TextStyle, ViewProps, ViewStyle } from 'react-native';
+
+import { LabelPosition } from '../Label/types';
+
+// This type should live elsewhere
+export type Sizes = 'small' | 'medium' | 'large';
+
+export interface RadioProps<T> extends ViewProps {
+  disabled?: boolean;
+  label?: string;
+  labelPosition?: LabelPosition;
+  labelStyle?: StyleProp<TextStyle>;
+  onChange?: (value?: T) => void;
+  radioButtonContainerStyle?: StyleProp<ViewStyle>;
+  radioButtonStyle?: StyleProp<ViewStyle>;
```

Comment:
Since these aren't really "button" styles, can we use names that are more indicative of their usage? Example:
```suggestion
  radioContainerStyle?: StyleProp<ViewStyle>;
  radioDotStyle?: StyleProp<ViewStyle>;
```